install.py

```diff
@@ -345,10 +345,7 @@ def validate_path(path):
     if cuda and curand_dir is not None:
         cmake_flags += ["-Dcunumeric_cuRAND_INCLUDE_DIR=%s" % curand_dir]
     if legate_dir:
-        # TODO: Undo this! Only setting temporarily so CI passes
-        if exists(join(legate_dir, "lib", "cmake", "legate_core")):
-            cmake_flags += ["-Dlegate_core_ROOT=%s" % legate_dir]
-        elif exists(join(legate_dir, "legate_core-config.cmake")):
+        if exists(join(legate_dir, "legate_core-config.cmake")):
```

Comment:
I think just
```
if legate_dir:
  cmake_flags += ["-Dlegate_core_ROOT=%s" % legate_dir]
```
We'll find out if we did it right once we see how CI works post-merge :-P